xbmc/platform/darwin/PlatformDarwin.cpp

```diff
@@ -22,3 +22,12 @@ bool CPlatformDarwin::InitStageOne()
 
   return true;
 }
+
+bool CPlatformDarwin::SupportsUserInstalledBinaryAddons()
+{
+#if defined(TARGET_DARWIN_IOS) || defined(TARGET_DARWIN_TVOS)
```

Comment:
Can just use TARGET_DARWIN_EMBEDDED which covers both

Reply:
Good to know. Do you want me to change that?

Reply:
I'd prefer yes. The other thing that isn't in my domain, there may want to be a check about jailbreak status possibly inside the ifdef. @sy6sy2 can maybe provide some insight to whether that's desired, as jailbroken devices don't necessarily have the same signing requirements

Reply:
I should say that comment isn't about you implementing it, more a ping to @sy6sy2 as he has the JB knowledge

Reply:
> Can just use TARGET_DARWIN_EMBEDDED which covers both

done